packages/adminrouter/buildinfo.json

```diff
@@ -8,9 +8,9 @@
       },
       "adminrouter": {
           "kind": "git",
-          "git": "https://github.com/dcos/adminrouter.git",
-          "ref": "66e17d71d009214164ba121b031e6a5668c6c283",
-          "ref_origin": "master"
+          "git": "https://github.com/malnick/adminrouter.git",
+          "ref": "61b4d661feac753ab2a1eadb6cce6d658133c57a",
+          "ref_origin": "malnick/agent_adminrouter"
```

Comment:
@spahl I noticed this was on `next`. We need to merge github.com/dcos/adminrouter/pull/9 and point this back to dcos/adminrouter `master`.